packages/synapse-react-client/src/synapse-queries/entity/useGetQueryResultBundle.ts

```diff
@@ -195,15 +200,15 @@ export function useGetQueryResultBundleWithAsyncStatus<
       return rowResult
     } else if (statsResult.status === 'error') {
       return statsResult
-    } else if (rowResult.status === 'loading') {
+    } else if (rowResult.isLoading) {
       // if either query is loading, return the loading status
       return rowResult
-    } else if (statsResult.status === 'loading') {
+    } else if (statsResult.isLoading) {
       return statsResult
     } else {
       // Otherwise, both queries are successful or idle, Merge the results into a single object
-      if (rowResult.status === 'idle') {
-        // If the row result is idle, apply the stats result last to override the idle status
+      if (rowResult.isPending) {
+        // If the row result is pending, apply the stats result last to override the pending status
```

Comment:
`'idle'` status was removed. `'pending'` is subtly different but for this purpose should work the same.